contracts/NftFactory.sol

```diff
@@ -27,8 +29,11 @@ contract NftFactory is UUPSUpgradableTemplate {
   uint8 private _lastNft;
   mapping(uint8 => address) private _factories;
   mapping(uint8 => uint256) private _prices;
+  mapping(uint8 => uint256) private _pricesInSeed;
 
   uint256 public proceedsBalance;
+  uint256 public seedProceedsBalance;
+  SeedToken public seedToken;
```

Comment:
Where do you initialize `seedToken`? I don't see it.

Reply:
That is why the test fails.
You can do it modifying the initializer
```
function initialize(address seed) public initializer {
    __UUPSUpgradableTemplate_init();
   require(seed.isContract(), "NftFactory: seed is not a contract");
   seedToken = SeedToken(seed);
  }
```

Reply:
Right, fixed